src/patternfly/layouts/Gallery/gallery.scss

```diff
@@ -1,8 +1,30 @@
 .pf-l-gallery {
   --pf-l-gallery--m-gutter--GridGap: var(--pf-global--gutter);
-  --pf-l-gallery--GridTemplateColumns: repeat(auto-fill, minmax(250px, 1fr)); // The 250px value is here just so that the grid has a width to use in the minmax() for laying out the gallery columns.
+  --pf-l-gallery--GridTemplateColumns--min: 250px;
+  --pf-l-gallery--GridTemplateColumns--min--core: var(--pf-l-gallery--GridTemplateColumns--min);
+  --pf-l-gallery--GridTemplateColumns: repeat(auto-fill, minmax(var(--pf-l-gallery--GridTemplateColumns--min--core), 1fr)); // The 250px value is here just so that the grid has a width to use in the minmax() for laying out the gallery columns.
   --pf-l-gallery--GridTemplateRows: auto;
 
+  @media screen and (min-width: $pf-global--breakpoint--sm) {
```

Comment:
WDYT creating an SCSS function for this MQ block since it's a repeatable thing and it would be easy to make a mistake either writing/reviewing it?

Reply:
@mcoker check it out!